### PR TITLE
Fix Coe Issues

### PR DESCRIPTION
**Pull request by rajobasu**

### CODE REVIEW

File: src/main/java/seedu/address/logic/filters/Filters.java [hunk 1]
```diff
@@ -21,23 +21,27 @@ public static AbstractFilter getCorrespondingFilter(String info) {
...
             return new EmailFilter(argumentMultimap.getValue(PREFIX_EMAIL).get());
         }
 
         if (argumentMultimap.getValue(PREFIX_PHONE).isPresent()) {
+            System.out.println(100);
             return new PhoneNumberFilter(argumentMultimap.getValue(PREFIX_PHONE).get());
         }
 
         if (argumentMultimap.getValue(PREFIX_COE_EXPIRY).isPresent()) {
+            System.out.println(100);
```
Comment (nighoggDatatype) [inline src/main/java/seedu/address/logic/filters/Filters.java:44]: Are these debug code left over?

File: src/main/java/seedu/address/logic/filters/Filters.java [hunk 2]
```diff
@@ -21,23 +21,27 @@ public static AbstractFilter getCorrespondingFilter(String info) {
         ArgumentMultimap argumentMultimap = ArgumentTokenizer.tokenize(" " + info + " ", PREFIX_NAME,
             PREFIX_EMAIL,
             PREFIX_PHONE,
-            PREFIX_ADDRESS, PREFIX_DOB, PREFIX_TAG, PREFIX_CARS_OWNED, PREFIX_CARS_PREFERRED);
+            PREFIX_ADDRESS, PREFIX_DOB, PREFIX_TAG, PREFIX_CARS_OWNED, PREFIX_CARS_PREFERRED, PREFIX_COE_EXPIRY);
 
 
-        if (argumentMultimap.getTotalSize() != 2) { // since there is also a dummy position :(
+        if (argumentMultimap.getTotalSize() > 2) { // since there is also a dummy position :(
+            System.out.println(argumentMultimap);
```
Comment (nighoggDatatype) [inline src/main/java/seedu/address/logic/filters/Filters.java:28]: Are we just leaving this debug code in?

Comment (nighoggDatatype) [reply src/main/java/seedu/address/logic/filters/Filters.java:28]: I mean I suppose we could, but I want to make sure, given the other debug code you left in

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 3]
```diff
@@ -17,5 +17,5 @@
     public static final Prefix PREFIX_AND = new Prefix("/AND");
     public static final Prefix PREFIX_OR = new Prefix("/OR");
     public static final Prefix PREFIX_NOT = new Prefix("/NOT");
-    public static final Prefix PREFIX_COE_EXPIRY = new Prefix("coe/");
+    public static final Prefix PREFIX_COE_EXPIRY = new Prefix("ex/");
```
Comment (nighoggDatatype) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:20]: Is this the solution? Could you explain how this solution works?

**Review by nighoggDatatype — COMMENTED** [review]
Can you please confirm which issue you are trying to fix?
Is it this one: [https://github.com/AY2021S2-CS2103T-W12-2/tp/issues/147](https://github.com/AY2021S2-CS2103T-W12-2/tp/issues/147)

File: src/main/java/seedu/address/logic/filters/Filters.java (hunk 2 above)
Comment (rajobasu) [reply src/main/java/seedu/address/logic/filters/Filters.java:28]: hey, my bad. have removed the debug codes

File: src/main/java/seedu/address/logic/filters/Filters.java (hunk 1 above)
Comment (rajobasu) [reply src/main/java/seedu/address/logic/filters/Filters.java:44]: hey, my bad, have removed the debug codes.

File: src/main/java/seedu/address/logic/parser/CliSyntax.java (hunk 3 above)
Comment (rajobasu) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java:20]: Ig we already discussed!

**Review by OhJunMing — APPROVED** [review]

**Pull request merged**